SonarQube.TeamBuild.PreProcessor/Roslyn/RoslynAnalyzerProvider.cs

```diff
@@ -18,13 +18,17 @@ namespace SonarQube.TeamBuild.PreProcessor.Roslyn
 {
     public class RoslynAnalyzerProvider : IAnalyzerProvider
     {
-        public const string RoslynCSharpFormatName = "roslyn-cs";
-        public const string RoslynCSharpRulesetFileName = "SonarQubeRoslyn-cs.ruleset";
+        public const string RoslynFormatNamePrefix = "roslyn-";
```

Comment:
it's slightly better to make these `Format` rather than a `Prefix`:
`roslyn-{0}`
`SonarQubeRoslyn-{0}`

and then later on use `string.Format(foo, Language)` 

Then it's clear that these constants are going to be used in a language-dependent way.